lib/matplotlib/style/core.py

```diff
@@ -99,20 +90,27 @@ def use(style):
     Parameters
     ----------
     style : str, dict, Path or list
-        A style specification. Valid options are:
 
-        +------+-------------------------------------------------------------+
-        | str  | The name of a style or a path/URL to a style file. For a    |
-        |      | list of available style names, see `.style.available`.      |
-        +------+-------------------------------------------------------------+
-        | dict | Dictionary with valid key/value pairs for                   |
-        |      | `matplotlib.rcParams`.                                      |
-        +------+-------------------------------------------------------------+
-        | Path | A path-like object which is a path to a style file.         |
-        +------+-------------------------------------------------------------+
-        | list | A list of style specifiers (str, Path or dict) applied from |
-        |      | first to last in the list.                                  |
-        +------+-------------------------------------------------------------+
+        A style specification.
+
+        - If a str, this can be one of the style names in `.style.available`
+          (a builtin style or a style installed in the user library path).
+
+          This can also be a dotted name of the form "package.name.style_name";
+          in that case, "package.name" should be an importable Python package
+          name, e.g. at ``/path/to/package/name/__init__.py``; the loaded style
+          file is ``/path/to/package/name/style_name.mplstyle``.
```

Comment:
Also don't bother with subpackages here:

```suggestion
          This can also be a dotted name of the form "package.style_name";
          in that case, "package" should be an importable Python package
          name, e.g. at ``/path/to/package/__init__.py``; the loaded style
          file is ``/path/to/package/style_name.mplstyle``.
```